src/actions/posts.js

```diff
@@ -601,12 +601,12 @@ export function getPostThread(rootId) {
     };
 }
 
-export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
+export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE, fetchRoot = true) {
```

Comment:
`fetchRoot` seems like a misnomer to me: should this parameter be called, `fetchThreads`?